### ISSUE TITLE
babel-node shouldn't be used in production

### ISSUE THREAD

**Issue by dortamiguel** [swarthout/feathers-apollo#16]
http://babeljs.io/docs/usage/cli/#babel-node

I think this can be improved in de package.json

**Comment by swarthout** [comment]
You are absolutely correct. Could you submit a pull request with this change? Thanks

**Comment by dortamiguel** [comment]
Yep, I'm working on this. When I have it I will make a pull request.

(Awesome work by the way!)
